### DISCUSSION TITLE
Awful performance with latest stable version

### DISCUSSION THREAD

**Discussion by Mozitux** [AzuraCast/AzuraCast#6863] [Q&A]
Hello, I have a problem that I cannot resolve despite spending a lot of time trying different settings.

With recent versions of Azuracast, the stream is regularly choppy while no CPU, latency or ram issues are present on the admin page.

I tried to completely reinstall Azuracast on a fresh version of Debian 12, to change provider, to buy a new server for my KVM hypervisor, but the problem keeps coming back.

Here is some technical information:

### Hardware:

**Hypervisor:** Proxmox 8.1
**CPU:** 8v CPU on 2 x E5-2609 v4
**RAM:** 16 GB (no ballooning)
**HDD:** 256 GB (SAS)
**Network:** 2GB/s down 1GB/s up

### Software:

**VM OS (only hypervisor VM for testing):** Debian 12
**Azuracast:** v0.19.4 Stable • Docker • PHP 8.2

### Logs:

[last_logs.txt](https://github.com/AzuraCast/AzuraCast/files/13929641/last_logs.txt)

If anyone has had this problem before and managed to fix it, can you help me?
**EDIT:** I tried the latest rolling release version but that didn't fix the problem

**Comment by fsan2val** [comment]
You shouldn't have any problems. I have azuracast with Ubuntu and nothing new. It may be your hosting provider, try another provider

**Reply by Mozitux** [reply]
I have already tried with different providers, the problem remains the same, on the same hardware and software configuration, my old station on version _v0.18.5 Stable • Docker • PHP 8.2_ works perfectly

**Comment by BusterNeece** [comment]
If by "awful performance" you mean that the audio sounds bad, but not that there are CPU or RAM problems, then that may have been fixed in our latest Rolling Release. We've been working with Liquidsoap to address some issues relating to crackles and pops in the audio, especially with shorter files, that may be affecting your installation too.

**Reply by Mozitux** [reply]
Indeed, it is during transitions between music and jingles that the problem appears and continues for a few tens of seconds with a choppy sound.

Even with the latest version of the rolling release, the problem is still present. Is this due to too many playlists? (42 for 688 medias files)
